behandlingslager/domene/src/main/java/no/nav/ung/sak/behandlingslager/behandling/vilkår/VilkårResultatRepository.java

```diff
@@ -137,7 +132,7 @@ private void kopieringPrecondition(Long fraBehandlingId, Long tilBehandlingId) {
      * Optimalisert spørring for å hente vilkårsresultater uten regelsporing.
      */
     public List<VilkårPeriodeResultatDto> hentVilkårResultater(Long behandlingId) {
-        String sql = "select vv.vilkar_type, vp.fom, vp.tom, nullif('-', vp.utfall) as utfall, nullif('-', vp.overstyrt_utfall) as overstyrt_utfall, vp.avslag_kode" +
+        String sql = "select vv.vilkar_type, vp.fom, vp.tom, nullif(vp.utfall, '-') as utfall, nullif(vp.overstyrt_utfall, '-') as overstyrt_utfall, vp.avslag_kode" +
```

Comment:
What? Korleis kan dette ha vore feil så lenge utan at nokon har merket det. Eg ser at den kun brukes i BehandlingDtoTjeneste i k9-sak så det er mulig at frontend ikkje bryr seg om resultatet her?

Reply:
ja jeg tror ikke det er i bruk. Tilsvarende for k9-sak. 
https://github.com/navikt/k9-sak/pull/11668

Kan evt høre med noen fra frontend

Reply:
Snakka med Hallvard det feltet er ikke i bruk